server/i18n/en.json

```diff
@@ -8776,7 +8776,11 @@
   },
   {
     "id": "model.config.is_valid.message_export.global_relay.customer_type.app_error",
-    "translation": "Message export GlobalRelaySettings.CustomerType must be set to one of either 'A9' or 'A10'."
+    "translation": "Message export GlobalRelaySettings.CustomerType must be set to one of either 'A9', 'A10' or 'CUSTOM."
```

Comment:
Unrelated comment: Do we support an inbucket export? At least the code does: https://github.com/mattermost/enterprise/blob/589937e32da440afc02a316398a706dd5debba69/message_export/global_relay_export/smtp.go#L40-L56

Reply:
I think that's only to make some unit tests work?

Reply:
Does it actually work in the product if configured via the config file directly?

Reply:
I don't see why it wouldn't work

Reply:
@BenCookie95 Would you please open a ticket to add inbucket to the UI, if the code supports it?